relay-server/src/metric_stats.rs

```diff
@@ -18,6 +21,14 @@ fn volume_metric_mri() -> MetricName {
         .clone()
 }
 
+fn cardinality_metric_mri() -> MetricName {
+    static CARDINALITY_METRIC_MRI: OnceLock<MetricName> = OnceLock::new();
+
+    CARDINALITY_METRIC_MRI
+        .get_or_init(|| "g:metric_stats/cardinality@none".into())
+        .clone()
+}
```

Comment:
Could we make this function a `const`? Resolving the return value at compile time allows removing the `OnceLock`.

Reply:
`Arc` does not have a `const` constructor.